pkg/operator/apiservices/sync_take_over_apiservices_oauth_apiserver.go

```diff
@@ -63,7 +64,8 @@ func syncManageAPIServicesController(deployer statemachine.Deployer, authOperato
 			return err
 		}
 		if !converged && !operator.Status.ManagingOAuthAPIServer {
-			syncContext.Recorder().Warning("PrereqNotReady", "the encryption deployer hasn't yet converged, retrying in 5 minutes")
+			// !converged will also be reported during the oauth apiserver installation
```

Comment:
here https://github.com/openshift/library-go/blob/master/pkg/operator/encryption/deployer/revisionedpod.go#L168

Reply:
alternatively, we could export `progressing` and simply not fire an event in that case.